Dockerfile

```
@@ -3,24 +3,24 @@ ARG PYTHON_REQUIREMENTS=requirements.txt
 ENV PYTHONUNBUFFERED=1
 ENV DOCKERIZE_VERSION v0.6.1
 
-ENV PG_MAJOR 13
+ENV PG_MAJOR 15
 
 RUN set -ex; \
     curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - ; \
     echo "deb http://apt.postgresql.org/pub/repos/apt/ buster-pgdg main" > /etc/apt/sources.list.d/pgdg.list; \
     apt-get update -y; \
-    apt-get install -y --no-install-recommends \
-        postgresql-13 postgresql-client-13 postgresql-contrib-13 \
+     apt-get install -y --no-install-recommends \
+        postgresql-15 postgresql-client-15 postgresql-contrib-15 \
     ; \
     apt-get clean; \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*; \
     :
 
 ENV PATH $PATH:/usr/lib/postgresql/$PG_MAJOR/bin
 
-ENV PGDATA /var/lib/postgresql/data
-
-VOLUME /var/lib/postgresql/data
+ENV PGDATA /var/lib/postgresql/mathesar
+VOLUME /etc/postgres/15/mathesar
```

```suggestion
VOLUME /etc/postgresql/15/mathesar
```
Also, why not mount `/etc/postgresql/` instead? I thought you were going to try that so we can avoid fiddling around with mounting different version numbers if we want to upgrade and so on.